From 127f271b56bd7ccb0f3ff88709ef0f67f5d688bb Mon Sep 17 00:00:00 2001
From: Christoph Schulz <develop@kristov.de>
Date: Sun, 19 Feb 2017 13:57:22 +0100
Subject: [PATCH] isdn4k-utils: pppdcapiplugin: fix capiconn message printed on
 incoming call

The indices for extracting the calling and called numbers have been swapped.
---
 pppdcapiplugin/capiconn.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pppdcapiplugin/capiconn.c b/pppdcapiplugin/capiconn.c
index ba983ae..9fd44d1 100644
--- a/pppdcapiplugin/capiconn.c
+++ b/pppdcapiplugin/capiconn.c
@@ -1085,8 +1085,8 @@ static void handle_incoming_call(capi_contr * card, _cmsg * cmsg)
 	(*cb->debugmsg)("incoming call contr=%d cip=%d %s -> %s", 
 			card->contrnr,
 			cmsg->CIPValue,
-			plcip->conninfo.callingnumber+2,
-			plcip->conninfo.callednumber+3);
+			plcip->conninfo.callingnumber+3,
+			plcip->conninfo.callednumber+2);
 
 	if (cb->incoming == 0)
 		goto ignore;
-- 
2.10.2

